Jellyfin.Server/Startup.cs

```diff
@@ -79,6 +81,13 @@ public void ConfigureServices(IServiceCollection services)
             var acceptJsonHeader = new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Json, 1.0);
             var acceptXmlHeader = new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Xml, 0.9);
             var acceptAnyHeader = new MediaTypeWithQualityHeaderValue("*/*", 0.8);
+            Func<IServiceProvider, HttpMessageHandler> eyeballsHttpClientHandlerDelegate = (_) => new SocketsHttpHandler()
+            {
+                AutomaticDecompression = DecompressionMethods.All,
+                RequestHeaderEncodingSelector = (_, _) => Encoding.UTF8,
+                ConnectCallback = HttpClientExtension.OnConnect
```

Comment:
Would it make sense to make it configurable?

Reply:
I didn't think so since the default client is not configurable and the callback needs to be set the HttpClientExtension, since it contains the happy eyeballs protocol.